src/components/popup/details/purposeList/purposeList.jsx

```diff
@@ -43,7 +44,7 @@ export default class PurposeList extends Component {
 						</ul>
 					</LocalLabel>
 				</div>
-				<a onClick={onBack}><LocalLabel localizeKey='back'>Customize how this data is used from the previous page</LocalLabel></a>
+				<a onClick={onBack} style={{color: theme.textLink}}>><LocalLabel localizeKey='back'>Customize how this data is used from the previous page</LocalLabel></a>
```

Comment:
Did you mean to add an extra `>` here?  Looks weird

Reply:
There are uses of `>` or `<` elsewhere in the code but looking at a locally running CMP makes me believe it's a typo.